main.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2022 Red Hat, Inc.
+// All Rights Reserved.
+//
+//    Licensed under the Apache License, Version 2.0 (the "License"); you may
+//    not use this file except in compliance with the License. You may obtain
+//    a copy of the License at
+//
+//         http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+//    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+//    License for the specific language governing permissions and limitations
+//    under the License.
+package main
+
+import (
+	"beatshipper/cmd/shipper"
+)
+
+func main() {
+	shipper.Execute()
```

Comment:
That function returns a possible error you are just ignoring. You probably would like to print the stack trace or something against one.

Reply:
You're right. I've added it. Thank you.